template/src/enhancers.js

```diff
@@ -1,5 +1,5 @@
 import { intl } from 'preact-i18n';
-import definition from './intl.json';
```

Comment:
we should update https://github.com/Zimbra/zm-x-zimlet-template-default/blob/master/template/.eslintrc.js#L50 also

Reply:
@silentsakky Done.